finality-aleph/src/sync/mod.rs

```diff
@@ -78,7 +78,7 @@ pub trait Justification: Clone + Send + Sync + Debug + 'static {
 
 /// A verifier of justifications.
 pub trait Verifier<J: Justification> {
-    type Error: Display;
+    type Error: Debug + Display;
```

Comment:
hmmmmmm, I think we want this to be only display to disallow using expects and unwraps in production code :P

Reply:
Eh, I don't feel this is a particularly strong argument. I would also prefer this to not be here, but if the lack makes testing much harder then imo it can stay.

Reply:
we're our worst enemies...
fixed